src/components/[guild]/collect/components/CollectNft.tsx

```diff
@@ -0,0 +1,153 @@
+import {
+  Flex,
+  Skeleton,
+  Stack,
+  Tag,
+  Tooltip,
+  useColorModeValue,
+} from "@chakra-ui/react"
+import { Chains } from "chains"
+import ConnectWalletButton from "components/[guild]/Requirements/components/GuildCheckout/components/buttons/ConnectWalletButton"
+import SwitchNetworkButton from "components/[guild]/Requirements/components/GuildCheckout/components/buttons/SwitchNetworkButton"
+import {
+  CapacityTag,
+  EndTimeTag,
+  StartTimeTag,
+  getTimeDiff,
+} from "components/[guild]/RolePlatforms/components/PlatformCard/components/AvailabilityTags"
+import CollectNftButton from "components/[guild]/collect/components/CollectNftButton"
+import { useCollectNftContext } from "components/[guild]/collect/components/CollectNftContext"
+import useGuild from "components/[guild]/hooks/useGuild"
+import CircleDivider from "components/common/CircleDivider"
+import useNftDetails from "../hooks/useNftDetails"
+import CollectNftFeesTable from "./CollectNftFeesTable"
+
+const availibiltyTagStyleProps = {
+  bgColor: "transparent",
+  fontSize: "sm",
+  fontWeight: "medium",
+  colorScheme: "purple",
+  p: 0,
+}
+
+const CollectNft = () => {
```

Comment:
No need to review this one, it's just a copy-paste from the old RequirementsCard component